aws-predictions/src/main/java/com/amplifyframework/predictions/aws/http/LivenessWebSocket.kt

```diff
@@ -76,6 +78,10 @@ internal class LivenessWebSocket(
     private val signer = AWSV4Signer()
     private var credentials: Credentials? = null
 
+    private val sdf = SimpleDateFormat("yyyymmddhhmmss", Locale.US)
+    internal var offset = 0L
+    internal var closeExpected = false
```

Comment:
call it `reconnectWithOffset` maybe; set to false afterwards so that we only reconnect once (safeguard)